apps/modernization-ui/src/pages/addPatient/AddPatient.tsx

```diff
@@ -100,8 +100,26 @@ const AddPatient = () => {
         setEntryState({ step: 'entry' });
     };
 
+    function isValidForm(obj: any) {
```

Comment:
Two things here;  
1. This function is not checking the validity of the form only that missing fields exists in the new patient.  At this point the form is already valid.  It may benefit from a different name. 
2. The function is always given a type of `NewPatientEntry` so `any` can be removed.

Reply:
Understood